src/main/java/com/sngular/annotation/processor/PactDslProcessor.java

```diff
@@ -38,20 +32,7 @@
 import com.sngular.annotation.processor.exception.PactProcessorException;
 import com.sngular.annotation.processor.exception.TemplateFactoryException;
 import com.sngular.annotation.processor.exception.TemplateGenerationException;
-import com.sngular.annotation.processor.mapping.BigDecimalMapping;
-import com.sngular.annotation.processor.mapping.BigIntegerMapping;
-import com.sngular.annotation.processor.mapping.BooleanMapping;
-import com.sngular.annotation.processor.mapping.ByteMapping;
-import com.sngular.annotation.processor.mapping.CharMapping;
-import com.sngular.annotation.processor.mapping.DateMapping;
-import com.sngular.annotation.processor.mapping.DoubleMapping;
-import com.sngular.annotation.processor.mapping.FloatMapping;
-import com.sngular.annotation.processor.mapping.IntegerMapping;
-import com.sngular.annotation.processor.mapping.LongMapping;
-import com.sngular.annotation.processor.mapping.ShortMapping;
-import com.sngular.annotation.processor.mapping.StringMapping;
-import com.sngular.annotation.processor.mapping.TypeMapping;
-import com.sngular.annotation.processor.mapping.ZonedDateTimeMapping;
+import com.sngular.annotation.processor.mapping.*;
```

Comment:
Is better to have individual imports that a "wild" ( * )
I believe you did not import the style code file